src/boundary_conditions.jl

```diff
@@ -159,25 +232,30 @@ function ChannelBCs(;  north = BoundaryCondition(Flux, nothing),
     return FieldBoundaryConditions(x, y, z)
 end
 
-
 #####
-##### Boundary conditions for solutions to systems of equations
+##### Boundary conditions for model solutions
 #####
 
 """
     SolutionBoundaryConditions(u, v, w, T, S)
 
-Construct a NamedTuple of boundary conditions for a system of
-equations with solution fields `u`, `v`, `w`, `T`, and `S`.
+Construct a `NamedTuple` of `FieldBoundaryConditions` for a model with solution fields
+`u`, `v`, `w`, `T`, and `S`.
 """
 SolutionBoundaryConditions(u, v, w, T, S) = (u=u, v=v, w=w, T=T, S=S)
 
 """
-    HorizontallyPeriodicSolutionBCs(u=HorizontallyPeriodicBCs, ...)
+    HorizontallyPeriodicSolutionBCs(u=HorizontallyPeriodicBCs(), ...)
 
-Construct a NamedTuple of boundary conditions for a horizontally-periodic
-system of equations with solution fields `u`, `v`, `w`, `T`, and `S`. Non-default
-boundary conditions for any field must be horizontally-periodic.
+Construct a `NamedTuple` of `FieldBoundaryConditions` for a horizontally-periodic model
+configuration with solution fields `u`, `v`, `w`, `T`, and `S`. Boundary conditions may be
+specified via keyword arguments.
+
+By default `HorizontallyPeriodicBCs` are applied to each field (except for w where
+`NoPenetration` boundary conditions are applied at the top and bottom).
+
+Any specified non-default boundary condition must be horizontally-periodic, i.e. you can
```

Comment:
Let's avoid saying "you"; instead we can say "only the top and bottom boundary conditions can be changed."